### PR TITLE
Legionnaire submission

### PR DESCRIPTION
**Pull request by Kairus101**
Now with 66.5% more heads!

### CODE REVIEW

**Comment by Kairus101** [comment]
I'm not sure if when merging this - if it it would overwrite the existing jungling code in community. I could consider it odd seeing as this is a branch from a slightly older commit of jungleLib.. I may do a pull from jungleLib and push when I get home to be sure. This will also go for parasite.

**Comment by eeyrjmr** [comment]
Apologies... Work has been beyond crazy (don't become an engineer...)
With junglelib in the main tree and teambot implemented I can work on this in isolation to detemine what is specific to this.

**Comment by Kairus101** [comment]
Cool, also, I am studying engineering, sucks to be us, eh? :/

File: bots/legionnaire/legionnaire_main.lua [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+
+		if #tResult > 0 then
+			return tResult
+		end
+	end
+
+	return nil
+end
+
+local function getTauntRadius()
```
Comment (Anakonda) [inline bots/legionnaire/legionnaire_main.lua:371]: Why is this function and not variable.

File: bots/legionnaire/legionnaire_main.lua [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+function HealAtWellUtilityOverride(botBrain)
+    return object.HealAtWellUtilityOld(botBrain)*1.75+(botBrain:GetGold()*8/2000)+ 8-(core.unitSelf:GetManaPercent()*8)
+end
+object.HealAtWellUtilityOld = behaviorLib.HealAtWellBehavior["Utility"]
+behaviorLib.HealAtWellBehavior["Utility"] = HealAtWellUtilityOverride
+
+-----------------------------------
+--          Custom Chat          --
+-----------------------------------
+
```
Comment (Anakonda) [inline bots/legionnaire/legionnaire_main.lua:753]: Chat messages into stringtables

**Comment by Anakonda** [comment]
He went into enemy jungle at 6min mark. Thats bad.

**Comment by Kairus101** [comment]
I wasn't sure whether that should be changed or not, it makes him more active, having to cut across mid-lane, and therefore actually ganking, I just didn't want a really passive, jungle-type hero is all.. From a strategy point of view, it's bad, but from a playing point of view, I prefer it, it's more fun.

**Pull request closed without merging**